$OpenBSD: patch-prnt_cupsext_cupsext_c,v 1.5 2015/04/17 04:18:13 ajacoutot Exp $

Backwards memcpy, triggered by changing default page size in hp-toolbox

--- prnt/cupsext/cupsext.c.orig	Tue Apr 14 11:52:35 2015
+++ prnt/cupsext/cupsext.c	Fri Apr 17 05:53:22 2015
@@ -1163,7 +1163,7 @@ PyObject * removeOption( PyObject * self, PyObject * a
 
             if ( j < g_num_options )
             {
-                memcpy( (g_options + j), (g_options + j + 1),
+                memmove( (g_options + j), (g_options + j + 1),
                         sizeof(cups_option_t) * (g_num_options - j) );
 
                 r = 1;
